docs_source_files/content/webdriver/driver_requirements.en.md

```diff
@@ -632,4 +632,25 @@ so whilst the web will continue to change, PhantomJS will not be updated.
 This was after Google announced the ability to run Chrome headlessly, 
 something also now offered by Mozilla's Firefox.
 
+### Chocolatey
```

Comment:
This section should be called in a way that reflects the content of simplifying the installation of drivers through package managers. It is important to add to the description of the section that the Selenium project does not maintain those packages, so no issues should be raised against this repo.